packages/webapp/components/layouts/AccountLayout/Security/index.tsx

```diff
@@ -100,10 +100,7 @@ function AccountSecurityDefault({
     AuthEvent.SocialRegistration,
     async (e) => {
       if (e.data?.flow) {
-        const flow = await getKratosSettingsFlow(
-          AuthFlow.Settings,
-          e.data.flow,
-        );
+        const flow = await getKratosFlow(AuthFlow.Settings, e.data.flow);
```

Comment:
Can you verify if this still works as intended? The issue here was, they have identical calls but the difference was the query parameters.

Reply:
Hmm weird, exact same type and call, but indeed made a difference!